hydromt_sfincs/plots.py

```diff
@@ -19,6 +19,7 @@
     "bnd": dict(marker="^", markersize=75, c="w", edgecolor="k", annotate=True),
     "src": dict(marker=">", markersize=75, c="w", edgecolor="k", annotate=True),
     "obs": dict(marker="d", markersize=75, c="w", edgecolor="r", annotate=True),
+    "crs": dict(linestyle=":", linewidth=1.0, color="k", annotate=False),
```

Comment:
this is a bit difficult to see with the dep file in the background. Perhaps use red color?

Reply:
Agree, do you want to stick to the ":" linestyle or are there also better options (exceptthe solid line I guess)? I often notice that dashed or dotted lines do not cover the entire line from start to end, because that may coincide with the "white spaces" in between the icons.